### ISSUE TITLE
Possible to run flatpak-builder-lint only to lint the metainfo?

### ISSUE THREAD

**Issue by cbm755** [flathub-infra/flatpak-builder-lint#295]
The [docs](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation) say this:
```
flatpak install -y flathub org.flatpak.Builder
flatpak run --command=appstream-util org.flatpak.Builder validate tld.domain.appid.metainfo.xml
```

But this seems to just be the standard appstream-util:
```
flatpak run --command=appstream-util org.flatpak.Builder --version
Version:	0.8.2
```
So it doesn't catch the extra things that this (new?) linter is catching (e.g. https://github.com/flathub/org.plomgrading.PlomClient/issues/110)

- - - -

So is it possible to run the new metadata validator "offline" (like without an entire build), just to validate the metainfo?  E.g., in my upstream CI.

**Comment by cbm755** [comment]
Somewhat answering my own question... I can do this:

```
flatpak run --command=flatpak-builder-lint org.flatpak.Builder manifest  org.plomgrading.PlomClient.yaml
```

So maybe this is really a feature request for:
```
flatpak run --command=flatpak-builder-lint org.flatpak.Builder metainfo org.foo.bar.metainfo.xml
```

**Comment by bbhtt** [comment]
The doc was updated a few hours ago https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation

Appstream-util is no longer used.


Anyway we were talking about this just today, so something like a subcommand should be made available.

**Comment by bbhtt** [comment]
Added in https://github.com/flathub-infra/flatpak-builder-lint/commit/c09ab63895e817d9214a0fb75afe21823f0f589a
